---
bool connected
bool armed
string mode
uint8 landed
